integration-tests/platform-http-proxy-ssl/src/main/resources/application.properties

```diff
@@ -16,4 +16,4 @@
 ## ---------------------------------------------------------------------------
 
 
-quarkus.native.resources.includes=certs/proxy-ssl-keystore.p12
\ No newline at end of file
+quarkus.native.resources.includes=target/certs/proxy-ssl-keystore.p12
```

Comment:
I guess we can delete this file is nothing is reading certs from the classpath anymore.

Reply:
fixed